src/Sylius/Bundle/ApiBundle/Resources/config/validation/AddressOrder.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                    xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
+
+    <class name="Sylius\Bundle\ApiBundle\Command\Checkout\AddressOrder">
+        <property name="billingAddress">
+            <constraint name="NotNull">
+                <option name="message">sylius.address.not_null</option>
+                <option name="groups">
+                    <value>sylius</value>
+                </option>
+            </constraint>
+            <constraint name="Sylius\Bundle\ApiBundle\Validator\Constraints\AddressingNotEmpty">
+                <option name="message">sylius.address.first_name.not_blank</option>
+                <option name="groups">
+                    <value>sylius</value>
+                </option>
+            </constraint>
+        </property>
```

Comment:
We already have some valiudation logic for addresses and we should use it here as well. Taking a look into `src/Sylius/Bundle/CoreBundle/Resources/config/validation/Order.xml` file, we should configure something like:
```suggestion
        <property name="billingAddress">
            <constraint name="Valid" />
        </property>
```